build.xml

```diff
@@ -20,25 +20,20 @@
         <mkdir dir="${module_build_path}" />
     </target>
 
-    <target name="install_js">
-        <echo msg="Installing JS dependencies" />
+    <target name="install_npm">
+        <echo msg="Installing NPM dependencies" />
         <exec command="npm install --no-bin-links" dir="${module_base_dir}" checkreturn="true" />
     </target>
 
-    <target name="maybe_install_js" unless="js_installed">
-        <phingcall target="install_js" />
+    <target name="maybe_install_npm" unless="npm_deps_installed">
+        <phingcall target="install_npm" />
     </target>
 
-    <target name="build_js_module">
-        <echo msg="Building JS module '${node_module_name}'" />
-        <exec command="npm run build -- --output-path='${module_build_path}/${node_module_name}'" dir="${node_modules_path}/${node_module_name}" />
+    <target name="build_css_module" depends="clean,prepare,maybe_install_npm">
```

Comment:
Why did you have to rename this? I've tried to keep all buildfiles more or less the same, so that changes can be ported easily, and new things can easily be added. Also, similar structure of buildfiles will not require a lot of learning to understand them. But I don't understand why you renamed things here. What is a "css module"? It's a JS module, you just build it how it knows to build its own CSS. NPM dependencies are JS dependencies. Please let's see how we can go back to a common format.